backend/db/utils.py

```diff
@@ -50,6 +51,24 @@
 SCHEMA = CONFIG["schema"]
 
 
```

Comment:
New route:
 - log-concepts

Reply:
Is this still being used, or part of an abandoned alternative method?